osu.Framework/Graphics/Colour/SRGBColour.cs

```diff
@@ -59,8 +72,8 @@ public struct SRGBColour : IEquatable<SRGBColour>
                 first.Linear.A + second.Linear.A),
         };
 
-        public readonly Vector4 ToVector() => new Vector4(Linear.R, Linear.G, Linear.B, Linear.A);
-        public static SRGBColour FromVector(Vector4 v) => new SRGBColour { Linear = new Color4(v.X, v.Y, v.Z, v.W) };
+        public readonly Vector4 ToVector() => new Vector4(SRGB.R, SRGB.G, SRGB.B, SRGB.A);
```

Comment:
The code quality failures here are likely worth considering. At very least we'd want to reduce the copy count to one by moving the property access to a local field, or at best avoid it being a computed property in the first place.

```suggestion
        public readonly Vector4 ToVector()
        {
            var linear = Linear;
            return new Vector4(linear.R, linear.G, linear.B, linear.A);
        }
```

Although I don't see this inspection locally so I'm not sure where it's coming from..

Reply:
I guess it's more of which is prominent in the codebase now (in terms of direct usages). I didn't notice the operators were widely used though.

Reply:
That said on second thought draw nodes are subject to invalidation, so the `ApplyChild()` calls should be 99% one-and-done init logic. So optimising for reads and avoiding `.ToLinear()` calls on the cpu *is* probably what we want.